f2/server.js

```diff
@@ -38,6 +38,42 @@ async function save(data, res) {
   saveBlob(data)
   data.submissionTime = new Date().toISOString()
 
+const url = `mongodb://${dbName}:${dbKey}@${dbName}.documents.azure.com:10255/mean-dev?ssl=true&sslverifycertificate=false`
+
+const uploadData = (req, res) => {
+  new formidable.IncomingForm().parse(req, (err, fields, files) => {
```

Comment:
Norm moved pretty much all of this code into their own files - which makes the code much cleaner but has led to serious merge conflicts with other PRs touching the server 🙁 It might be easier to just close this PR and make a new one with the important bits you figured out added to `scanFiles.js`
```
if (file.size > 4194304) {
        res.statusCode = 400
        res.statusMessage =
          'Warning: One of the submitted files is bigger than 4MB.'
        res.send(res.statusMessage)
        return
      }
```

Reply:
Ohh I see, Ok that makes sense... I'll close this.